src/stream/stream_backend.rs

```diff
@@ -8,7 +8,10 @@ use crate::video::types::{VideoEncodeType, VideoSourceType};
 use crate::video_stream::types::VideoAndStreamInformation;
 use simple_error::{simple_error, SimpleError};
 
-pub trait StreamBackend {
+pub trait StreamBackend
+where
+    Self: Drop,
```

Comment:
What about the warning ?
```
bounds on `Self: std::ops::Drop` are most likely incorrect, consider instead using `std::mem::needs_drop` to detect whether a type can be trivially dropped
```

Reply:
From my understanding, it is just saying that we might don't need drop because the types would that implement StreamBackend trait might have trivial drop.

Reply:
Please, if possible, it would be nice to add `#[allow(drop_bounds)]`